prediction_market_agent_tooling/tools/utils.py

```diff
@@ -173,3 +174,35 @@ def prob_uncertainty(prob: Probability) -> float:
         - Market's probability is 0.95, so the market is quite certain about YES: prob_uncertainty(0.95) == 0.286
     """
     return float(entropy([prob, 1 - prob], base=2))
+
+
+def calculate_sell_amount_in_collateral(
+    shares_to_sell: float,
+    holdings: float,
+    other_holdings: float,
+    fee: float,
+) -> float:
+    """
+    Computes the amount of collateral that needs to be sold to get `shares`
+    amount of shares. Returns None if the amount can't be computed.
+
+    Taken from https://github.com/protofire/omen-exchange/blob/29d0ab16bdafa5cc0d37933c1c7608a055400c73/app/src/util/tools/fpmm/trading/index.ts#L99
+    Simplified for binary markets.
+    """
+
+    if fee >= 1 or fee < 0:
```

Comment:
```suggestion
    if not (0 < fee <= 1.0):
```

Just a small python sugar trick, no need to commit 😄